recipes-core/dist-nilrt/files/nilrt-gateway-install

```diff
@@ -38,29 +38,36 @@ function die() {
     exit 1
 }
 
-# Download dist-nilrt-efi-ab.ipk into /payload of nirootfs partition
-NIROOTFS_MOUNT=$(mktemp -d "/var/volatile/niroot-XXXXXXX")
-mount -L nirootfs $NIROOTFS_MOUNT || die "Unable to mount nirootfs."
-
-rm -rf $NIROOTFS_MOUNT/payload
-mkdir $NIROOTFS_MOUNT/payload || die "Unable to create payload directory."
-
-pushd $NIROOTFS_MOUNT/payload > /dev/null
-opkg update > /dev/null
-opkg download dist-nilrt-efi-ab > /dev/null || (popd > /dev/null; die "Unable to download dist-nilrt-efi-ab. Check if disk has sufficient space.")
-test -e dist-nilrt-efi-ab*.ipk || (popd > /dev/null; die "Unable to download dist-nilrt-efi-ab. Check if feeds contain dist-nilrt-efi-ab package.")
-popd > /dev/null
-
 # Mount the ISO/wic file as a loopback device
 info "Mounting the ISO"
 if [ -f "$SCRIPT_DIR/restore-mode-image-x64.iso" ] ; then
+    # safemode->efi-ab migration
```

Comment:
NIT: let's prefer `efi-grub` to `safemode` in these comments; it will make more sense to the future maintainers and community, I hope.

Reply:
Good point, thanks, it's changed.